Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -762,6 +764,14 @@ def expanded(feat):
         indices = inst_by_id[inst_id]
         instance = newtable[i]
 
+        for var in domain.variables + domain.metas:
+            if var in idvarlist:
+                continue
+            if numpy.isnan(instance[var]):
+                for ind in indices:
```

Comment:
Why loop so explicitly? Doesn't table handle advanced indexing and broadcasting?